dlion168/learning-to-read-chinese
# TeachWordView setLearned

TBD
- update bottom page immediately (isLearned)
- disable tab
- wait till voice ends
- use device height to set size
- practice with/without border 防呆
---
(@dlion168)
- db getPhraseId error
- progressBar
- update lastest bpmf quiz code
---
- check img, Word size/length
- check if bopomofo sentence has >1 vocab in it
- modify all to Navigator.of(context).push(MaterialPageRoute())
- svg stroke
